### PR TITLE
Task 52 / Change json flag

### PR DESCRIPTION
**Pull request by grananda**
JSON option flag was modified by --output in order to allow multiple output parameters in the futute.

### CODE REVIEW

File: src/main/java/io/seqera/tower/cli/Tower.java [hunk 1]
```diff
@@ -67,8 +67,8 @@
     @Option(names = {"-u", "--url"}, description = "Tower server API endpoint URL. Defaults to tower.nf (TOWER_API_ENDPOINT)", defaultValue = "${TOWER_API_ENDPOINT:-https://api.tower.nf}", required = true)
     public String url;
 
-    @Option(names = {"-j", "--json"}, description = "Show output as JSON")
-    public boolean json;
+    @Option(names = {"--output"}, description = "Show output in defined format (only JSON is available so far)")
+    public String output;
```
Comment (jordeu) [inline src/main/java/io/seqera/tower/cli/Tower.java:71]: Use an enum here. In that way picocli validates the value.

File: src/main/java/io/seqera/tower/cli/Tower.java [hunk 2]
```diff
@@ -67,8 +67,8 @@
     @Option(names = {"-u", "--url"}, description = "Tower server API endpoint URL. Defaults to tower.nf (TOWER_API_ENDPOINT)", defaultValue = "${TOWER_API_ENDPOINT:-https://api.tower.nf}", required = true)
     public String url;
 
-    @Option(names = {"-j", "--json"}, description = "Show output as JSON")
-    public boolean json;
+    @Option(names = {"--output"}, description = "Show output in defined format (only JSON is available so far)")
```
Comment (jordeu) [inline src/main/java/io/seqera/tower/cli/Tower.java:70]: Improve the description, it is not clear if the user has to pass `--output JSON`  or `--output json`.

**Review by jordeu — CHANGES_REQUESTED** [review]

File: src/main/java/io/seqera/tower/cli/commands/AbstractApiCmd.java [hunk 3]
```diff
@@ -260,7 +261,7 @@ protected String workspaceRef(Long workspaceId) throws ApiException {
     public Integer call() {
         try {
             Response response = exec();
-            if (app().json) {
+            if (app().output != null && Objects.equals(app().output, OutputType.json)) {
```
Comment (jordeu) [inline src/main/java/io/seqera/tower/cli/commands/AbstractApiCmd.java:264]: I think this can be simplify to only `app().output == OutputType.json`

Comment (grananda) [reply src/main/java/io/seqera/tower/cli/commands/AbstractApiCmd.java:264]: It was a fancy Intellij suggestion... it pisses me off when the IDE underlines the code in orange.. will revert :)

**Review by jordeu — APPROVED** [review]

**Pull request merged**